server/models/index.js

```diff
@@ -1,36 +1,34 @@
-const { Photo } = require('../../database/schema.js');
+const { Rooms, Photos } = require('../../database/postgres/models/index.js');
 
-const read = (params) => {
-  return Photo
-    .find(params)
-    .exec()
-    .then(photos => photos)
+const read = async (params) => {
```

Comment:
since you are returning a promise does this need to be an async function?

Reply:
Thank you for catching this, Danny. I have removed the unnecessary async.